WooCommerce/Classes/View Modifiers/WooStyleModifiers.swift

```diff
@@ -86,6 +86,13 @@ struct ErrorStyle: ViewModifier {
     }
 }
 
+struct NavigationBarStyle: ViewModifier {
```

Comment:
What do you think about naming this something like `AccentBarStyle` or `AccentNavigationBarStyle`.

If I see `NavigationBarStyle` in the wild I would not know what style I am applying 😅

Reply:
I named it `NavigationBarStyle` with the idea that, if needed, other styling could be added in the future that also applies to all navigation bars / elements in the navigation bar. Along those lines, I like your suggestion below to use `wooNavigationBarStyle` so maybe this can be `WooNavigationBarStyle` — since this is used to add the default Woo styling to the navigation bar.